blackbox/docs/protocols/http.txt

```diff
@@ -93,7 +95,8 @@ It is possible to set a default schema while querying the Crate cluster via
 ``_sql`` end point. In such case the HTTP request should contain the
 ``Default-Schema`` header with the specified schema name::
 
-    sh$ curl -sSXPOST '127.0.0.1:4200/_sql?pretty' -H 'Default-Schema: doc' -d '{
+    sh$ curl -H 'Content-Type: application/json' \
+    ... -sSXPOST '127.0.0.1:4200/_sql?pretty' -H 'Default-Schema: doc' -d '{
```

Comment:
should we use heredocs here as well? just to be consistent?
also further down this document.